lib/analysis/offline/spider/SpiderAnalyzer.ts

```diff
@@ -88,9 +94,27 @@ async function extractAtomicAspects(p: Project,
 }
 
 async function extractify(aspect: Aspect, p: Project, timeRecorder: TimeRecorder): Promise<FP[]> {
+    const minimalPushImpactListenerInvocation: PushImpactListenerInvocation = {
```

Comment:
is it gonna be possible later to implement these methods?

Reply:
Probably not. The addition of the `PushImpactListenerInvocation` argument isn't very friendly to spidering. It might be possible to omit the impossible-to-implement methods in the type of the argument to `extract`, but that's in the fingerprints pack.